### PR TITLE
Fixes #33038 - add bmc_available to host show API

### PR DESCRIPTION
**Pull request by amirfefer**
This is needed for the new host details page to enable  BMC power actions

### CODE REVIEW

File: app/views/api/v2/hosts/main.json.rabl [hunk 1]
```diff
@@ -16,7 +16,7 @@ attributes :ip, :ip6, :environment_id, :environment_name, :last_report, :mac, :r
   :compute_resource_id, :compute_resource_name,
   :compute_profile_id, :compute_profile_name, :capabilities, :provision_method,
   :certname, :image_id, :image_name, :created_at, :updated_at,
-  :last_compile, :global_status, :global_status_label, :uptime_seconds, :tags
+  :last_compile, :global_status, :global_status_label, :uptime_seconds, :tags, :bmc_available?
```
Comment (ares) [inline app/views/api/v2/hosts/main.json.rabl]: Just a tiny ask, don't add an attribute with `?` in the name.

While it's Rails convention, the API shouldn't contain special charatcters, we typically don't do it for other flags, such as managed or build or enabled. Could you pleaes define an alias in the host model, something like `alias_method :bmc_available?, :bmc_available` and then use the version without question mark here?

**Review by ares — CHANGES_REQUESTED** [review]
Tiny change requested, then good to be merged.

**Comment by amirfefer** [comment]
Thanks @ares , updated

**Review by ares — APPROVED** [review]
Thanks @amirfefer, works well, merging!

**Pull request merged**